### DISCUSSION TITLE
Welcome to friendly-traceback Discussions!

### DISCUSSION THREAD

**Discussion by aroberge** [aroberge/friendly#127] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RichardHack** [comment]
I am a  computer teacher in high school.I need this to help me learn Python better especially when i get into some hard problems.

**Reply by aroberge** [reply]
I hope you do find it useful. You may want to consult the documentation: https://aroberge.github.io/friendly-traceback-docs/docs/html/

Feel free to ask any questions if you need help on how to use Friendly-traceback.
